packages/material-ui/src/Select/SelectInput.js

```diff
@@ -276,8 +250,10 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
         aria-haspopup="listbox"
         aria-owns={open ? `menu-${name || ''}` : undefined}
         onKeyDown={handleKeyDown}
-        onBlur={handleBlur}
         onClick={disabled || readOnly ? null : handleClick}
+        /* prevent focusing  the trigger since we'll open anyway and move focus */
+        onMouseDown={e => e.preventDefault()}
```

Comment:
It simply looked weird if we focus the input for a short period since the focus style is so prominent (shrunk label). Previously this was kind of hacky since focused styles were still applied when the select was open even though it wasn't focused anymore.

I think the biggest problem is that the full <Input /> component is used for this component. It smells like over abstraction though I'm likely missing something. I would hope this component can become a lot smaller and easier to navigate by just copying the relevant parts from Input.

Reply:
We want to display the select in active state since this is closer to the native select but does not follow WAI-ARIA.